server/channels/app/notification.go

```diff
@@ -1465,7 +1465,7 @@ func (a *App) allowChannelMentions(c request.CTX, post *model.Post, numProfiles
 
 // allowGroupMentions returns whether or not the group mentions are allowed for the given post.
 func (a *App) allowGroupMentions(c request.CTX, post *model.Post) bool {
-	if license := a.Srv().License(); license == nil || (license.SkuShortName != model.LicenseShortSkuProfessional && license.SkuShortName != model.LicenseShortSkuEnterprise) {
+	if license := a.Srv().License(); license == nil || !model.MinimumProfessionalLicense(license) {
```

Comment:
Same nit here about nil check
```suggestion
	if !model.MinimumProfessionalLicense(a.Srv().License()) {
```

Reply:
Done.